snpm_cp.m

```diff
@@ -782,6 +798,45 @@ function snpm_cp(CWD)
 %-Loop over planes (breaks out after first loop if bVolm)
 %-----------------------------------------------------------------------
 nP = [];
+
+if(UseRapidPT >= 1)
```

Comment:
So is `bhPerms` used at all?  SnPM constructs a `MaxT` that has first column that is max T stat and second column that is *minus* the the min T. As a two-sample t-test does not assume distributional symmetry, you wouldn't necessarily want to automatically merge these two columns.  Only under certain settings, specifically when all permutations are exhaustively computed does SnPM set `bhPerm=1`.

So when `bhPerm=0`, MaxT(:,1) is only used to make inferences on increases, MaxT(:,1) is only used to make inferences on decreases (multiplied by -1).  When`bhPerm=1`, we allow ourselves to construct the max T distribution out of MaxT(:).

I assume you probably don't want to assume distributional symmetry (and you're not exhustive), so you should set `bhPerm=0` at the top of this RapidPT code block.